lib/httplib/httplib.go

```diff
@@ -84,9 +84,8 @@ func WithCSRFProtection(fn HandlerFunc) httprouter.Handle {
 		err := csrf.VerifyHTTPHeader(r)
 		if err != nil {
 			log.Warningf("unable to validate CSRF token %v", err)
-			terr := trace.AccessDenied("access denied")
-			log.Warnf("%v", trace.DebugReport(terr))
-			trace.WriteError(w, trace.Unwrap(terr))
+			log.Warnf("%v", trace.DebugReport(err))
```

Comment:
nit: historically, the output of `trace.DebugReport` was escaped when using the trace's `TextFormatter`. We came up with a hack to avoid the escaping by wrapping the error with `logrus.WithError` so it is output w/o escaping to improve readability.